packages/docs/docs/docs/upgrading.md

```diff
@@ -35,11 +35,18 @@ just as with any other contract on the blockchain, we can define governance
 mechanisms to decide when and how to upgrade the contracts, that can be manual,
 automated, or any combination of both that will earn the trust of our users.
 
-Let's create an upgradeable instance of this contract so you can experiment what
-this is all about:
+We also started a session to work with the `local` network, before we continue, 
+let's renew the session to ensure it is not expired:
 
 ```console
-zos create MyContract --init initialize --args 42,hitchhiker --network local
+zos session -n local -f 0x1df62f291b2e969fb0849d99d9ce41e2f137006e --expires 3600
+```
+
+Great, now let's create an upgradeable instance of this contract so you can 
```

Comment:
-1 to `Great` and similar expressions, but it's because I'm a boring guy. If @elopio approved it, then it's ok with me.